fastlane/Fastfile

```diff
@@ -108,7 +108,7 @@ platform :android do
       android_artifact_type: "APK",
       android_artifact_path: "app/build/outputs/apk/release/com.bitwarden.authenticator-release.apk",
       service_credentials_file: options[:serviceCredentialsFile],
-      groups: "internal-prod-group",
+      groups: "internal-prod-group, livefront",
       release_notes: release_notes,
     )
   end
```

Comment:
I'm thinking about removing this entire call, since it is an APK upload and is redundant with the aar upload.

Reply:
I'll do this as a separate PR!